### DISCUSSION TITLE
Can I call Orchid's fields inside blade template?

### DISCUSSION THREAD

**Discussion by itpmsoftware** [orchidsoftware/platform#2386] [Q&A] [answered]
Hi, I'm trying to call a component (Input or Select) inside the blade template, I tried to call with <x-orchid-input>, I'm a new developer using laravel and orchid, and I don't know if it's possible or not

**Comment by tabuna** [comment, marked as answer]
In the `blade` view, you can use it like this:

```php
{!!  \Orchid\Screen\Fields\Input::make('email')
    ->type('email')
    ->required()
    ->tabindex(1)
    ->autofocus()
    ->placeholder(__('Enter your email'))
!!}
```

**Reply by itpmsoftware** [reply]
Thanks bro!
